scripts/core/superdesk-api.d.ts

```diff
@@ -903,6 +903,7 @@ declare module 'superdesk-api' {
         saml_auth: any;
         google_auth: any;
         saml_label: any;
+        slugline_autocomplete: boolean;
```

Comment:
Features should not be associated with specific field types. If you want to display whitespace for `body_html` field, you don't tie it with `body_html` but implement it in editor3 and all fields can use it. Same here. The implementation of autocomplete for `slugline` is the same as it would be for any other plain text field.

Can you make it generic & configurable for any plain text field?

Reply:
would there be a usecase for another field @karelpetrak ?

Reply:
It's not only about future requirements but about code quality too. If there aren't functional differences it shouldn't be a problem to make it work for all text fields. If it is a problem, it's a good opportunity to discover and improve architectural issues.